docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -2404,7 +2404,7 @@ Since appointments have no draft state, if `saveAsync` is called on an appointme
 > [!NOTE]
 > The following clients have different behavior for `saveAsync` on appointments in compose mode:
 >
-> - Mac Outlook does not support `saveAsync` on a meeting in compose mode. Calling `saveAsync` on a meeting in Mac Outlook will return an error.
+> - Outlook for Mac does not support `saveAsync` on a meeting in compose mode. As such, calling `saveAsync` in that scenario returns an error. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.
```

Comment:
I recommend using this text from another PR you created. I like this wording better. I think it reads easier.

Outlook for Mac does not support saving a meeting. The `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround.

Reply:
I recommend using this wording through the rest of the files in this PR.